arviz/data/inference_data.py

```diff
@@ -326,18 +328,35 @@ def from_netcdf(filename: str) -> "InferenceData":
         ----------
         filename : str
             location of netcdf file
+        group_kwargs : dict of dict
+            Keyword arguments to be passed into each call of `xarray.open_dataset`.
+        regex : str
```

Comment:
```suggestion
        regex : bool, default False
```

I think this would be the correct docstring for this parameter